### ISSUE TITLE
unauthenticated function does not trigger

### ISSUE THREAD

**Issue by ichuprov** [rappasoft/laravel-5-boilerplate#950]
Using Laravel 5.4 Boilerplate

I'm adding in a new route
```
Route::group(['middleware' => 'auth'], function () {
    Route::get('project/add', 'Project\ProjectController@create')->name('project.add');
});
```

Problem is when I access `/project/add` I get the authentication exception which doesn't trigger the unauthenticated function which should redirect to the Login page. The unauthenticated function never triggers.

I even did a test on a preset route (that came with the boilerplate) eg `/dashboard`
Same thing happens. Instead of redirecting to the Login page, it displays the authentication exception error on screen.
AuthenticationException is added to the do not report array in Exceptions/handler.php

Is there a reason why the redirect would not be working?

**Comment by Shuyinsama** [comment]
I can't seem to reproduce this issue with a new installation.
Do you have logs of this inside `storage/logs` ?

Or would it be possible to upload a use-case with this error so I could test this?

Thanks in advance

**Comment by ichuprov** [comment]
Well nothing gets logged because Handler is told not to report it.
I just get the on screen Exception displayed.
It's very weird that unauthenticated just does not get triggered at all.

Not sure about a use case because it happens for any route that uses 'auth' middleware.

To work around this, I've taken it out of the dontReport array, and added a check in the render function. This now redirects when AuthenticationException occurs.

**Comment by Shuyinsama** [comment]
As a note: We just released the 5.5 update on the master branch so maybe that will solve this issue.

But seeing as it's not always easy to update I would like to still help you out. 
I just tried a fresh install of the 5.4 branch too and I still can't seem to replicate this problem on my macbook. So that leads me to believe it's an environment issue.

What PHP version and what is the server environment like?
